apps/web/src/services/asset-inventory/collector/collector-create/modules/MultipleProviderForm.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+    <div class="multiple-provider-form">
+        <p-field-title class="title">
+            Provider
+        </p-field-title>
+        <div class="radio-container">
+            <p-radio-group>
+                <p-radio v-for="provider in state.providerList"
+                         :key="provider.name"
+                         :selected="state.selectedProvider"
+                         :value="provider.name"
+                         class="provider-item"
+                         @change="handleChangeProvider"
+                >
+                    <div class="content-menu-item">
+                        {{ provider.label }}
+                    </div>
+                </p-radio>
+            </p-radio-group>
+        </div>
+        <!-- MEMO: screen mobile size-->
+        <div class="dropdown-container">
+            <p-select-dropdown :selected="state.selectedProvider"
+                               :items="state.providerList"
+                               class="select-dropdown"
+                               @update:selected="handleChangeProvider"
+            >
+                <template #menu-item--format="{ item }">
+                    <div class="content-menu-item">
+                        <span>{{ item.label }}</span>
+                    </div>
+                </template>
+            </p-select-dropdown>
+        </div>
+    </div>
+</template>
+
+<script lang="ts" setup>
+import { computed, reactive } from 'vue';
+
+import {
+    PFieldTitle, PRadioGroup, PRadio, PSelectDropdown,
+} from '@spaceone/design-system';
+
+
+import { store } from '@/store';
+
+import type { ProviderReferenceMap } from '@/store/modules/reference/provider/type';
+
+import {
+    useCollectorFormStore,
+} from '@/services/asset-inventory/collector/shared/collector-forms/collector-form-store';
+
+const collectorFormStore = useCollectorFormStore();
+const collectorFormState = collectorFormStore.$state;
```

Comment:
suggestion: Please move `MultipleProviderForm.vue` to under `shared/collector-form/` for consistency!

Reply:
This component is not a shared component!